### PR TITLE
attach files in description

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>

* Resolves: #1935
* Target version: master 

grafik

### CODE REVIEW

File: src/components/AttachmentDragAndDrop.vue [hunk 1]
```diff
@@ -207,8 +207,4 @@ export default {
 		margin: 20px 20px 60px 20px;
 	}
 
-	.modal__content button {
```
Comment (juliushaertl) [inline src/components/AttachmentDragAndDrop.vue:210]: Any reason to remove that? Seems unrelated.

Comment (jakobroehrl) [reply src/components/AttachmentDragAndDrop.vue:210]: The "upload attachment" Button is bad formatted if this is not removed

**Comment by juliushaertl** [comment]
@jakobroehrl "4. to release" is for approved ones that just need merging e.g. because ci is still pending. :wink:
